lib/send_grid_mailer/dev_deliverer.rb

```diff
@@ -24,7 +25,13 @@ def api_key
     end
 
     def letter_opener_delivery_method
-      @letter_opener_delivery_method ||= LetterOpener::DeliveryMethod.new(location: '/tmp/mails')
+      @letter_opener_delivery_method ||= LetterOpener::DeliveryMethod.new(location: dev_emails_location)
+    end
+
+    def dev_emails_location
+      Rails.application.config.action_mailer.sendgrid_dev_settings[:emails_location] || "/tmp/mails"
```

Comment:
under what conditions does this raise an exception? im personally not a fan of exceptions as control flow and always prefer checking things first. and also considering the `/tmp/mails` appearing twice, it looks weird to me

Reply:
If the config is not set, its value will be `nil` and the method will return "/tmp/mails" on line 32 (no error). I added the `rescue` to maintain the format used by another method on this file, which also reads a config.